input/pages/usecases.xml

```diff
@@ -403,7 +403,15 @@ code=69548-6&_include=Observation:
   <h4>Genomic Study</h4>
   <p>Users can query Genomic Study and Genomic Study Analysis instances. Use cases include identifying all genomic tests a patient has had, 
 determining the scope of a genomic study (e.g. genes studied, types of molecular alterations detectable), identifying those tests that used a particular specimen or occurred within a given time frame, etc. </p>
-  <p>The FHIR R4 backport of the FHIR R5/R6 GenomicStudy resource implements a number of R5/R6 attributes as Procedure extensions. Thus, much of the semantic content of a FHIR R4 Genomic Study is conveyed as extension values. Searching against extension values is complex, and a full description is outside the scope of this IG. We refer the reader to a <a href="https://stackoverflow.com/questions/53482491/fhir-search-by-extension">post by Lloyd McKenzie from Nov 2018</a> in which he states that "two steps are required: first, you need to define a custom SearchParameter that searchs on the element you wish in the way you wish. Second, all relevant servers must be manually changed (i.e. have code written) to support the new search parameter".</p>
+  <p>The FHIR R4 backport of the FHIR R5 GenomicStudy resource implements a number of R5 attributes as Procedure extensions. 
+  Thus, much of the semantic content of a FHIR R4 Genomic Study is conveyed as extension values. Searching against extension values is 
+  complex, and a full description is outside the scope of this IG. 
+  We refer the reader to a <a href="https://stackoverflow.com/questions/53482491/fhir-search-by-extension">post by Lloyd McKenzie 
```

Comment:
I am still not sure about having a link to StackOverflow from our IG. I am not sure we have to cite our source :) , so I wonder if we can just say the two steps that are required and say the implementation is left to the implementer for now?

Reply:
ok, I removed the link to StackOverflow and left in the two steps. Thanks.